docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -223,8 +223,28 @@ children:
     children:
 
     - title: 'Authentication'
-      url: Loopback-component-authentication.html
+      url: Authentication-overview.html
       output: 'web, pdf'
+      children:
+        - title: 'Apply JWT Authentication in Todo Example'
+          url: Authentication-tutorial.html
+          output: 'web, pdf'
+
+        - title: 'Authentication Component'
+          url: Loopback-component-authentication.html
+          output: 'web, pdf'
+
+        - title: 'JWT Authentication Extension'
+          url: JWT-authentication-extension.html
+          output: 'web, pdf'
+
+        - title: 'Implement your own authentication strategy'
+          url: Implement-your-own-strategy.html
+          output: 'web, pdf'
+
+        - title: 'Passport Adapter for Authentication'
```

Comment:
Note: use 
https://loopback.io/doc/en/lb4/Loopback-component-authentication.html#using-passport-based-strategies
and
https://github.com/strongloop/loopback-next/tree/master/extensions/authentication-passport